src/util/required-opts.js

```diff
@@ -0,0 +1,12 @@
+var _ = require('underscore');
+
+module.exports = function checkAndBuildRequiredOpts (actualOpts, requiredOpts, context) {
```

Comment:
* We already had an [util](https://github.com/CartoDB/cartodb.js/blob/v4-next/src/core/util.js) (by the way with outdated methods should we merge them?

* Should we use this check in the other Models? (ie: [CartoDBLayer](https://github.com/CartoDB/cartodb.js/blob/v4-next/src/geo/map/cartodb-layer.js#L21) )

Reply:
We can move it to `util`, yes.

My advice is to use wherever we can. It eases the `initialize` checks.